### DISCUSSION TITLE
numberoflines

### DISCUSSION THREAD

**Discussion by megabeehothanhminh** [MobileReality/react-native-select-pro#95] [General]
Hello. My option label is long. I want set numberOfLines for option item label. How can I do this?
Thank!

**Comment by megabeehothanhminh** [comment]
<Text numberOfLines={2} style={[styles.text, optionTextStyle]}>
                    {label}
</Text>

**Comment by irekrog** [comment]
Hi @megabeehothanhminh 

At this moment you have two options:
* use `patch-package` and override our code changing `numberOfLines`
* use a [OptionComponent](https://mobilereality.github.io/react-native-select-pro/docs/api/components#optioncomponent) prop and create own component for the option with your requirements
